include/opt-sched/Scheduler/sched_region.h

```diff
@@ -67,6 +67,9 @@ class SchedRegion {
   // Get the number of simulated spills code added for this block.
   inline int GetSimSpills() { return totalSimSpills_; }
 
+  // Gets the un-normalized incremental RP cost for the region(used by ACO)
+  virtual InstCount getUnnormalizedIncrementalRPCost() = 0;
```

Comment:
```suggestion
  virtual InstCount getUnnormalizedIncrementalRPCost() const = 0;
```

Should be `const`-correct

Reply:
did it